docs/settings.md

```diff
@@ -69,6 +69,7 @@ Using Uvicorn with watchfiles will enable the following options (which are other
 * `--http <str>` - Set the HTTP protocol implementation. The httptools implementation provides greater performance, but it not compatible with PyPy. **Options:** *'auto', 'h11', 'httptools'.* **Default:** *'auto'*.
 * `--ws <str>` - Set the WebSockets protocol implementation. Either of the `websockets` and `wsproto` packages are supported. Use `'none'` to ignore all websocket requests. **Options:** *'auto', 'none', 'websockets', 'wsproto'.* **Default:** *'auto'*.
 * `--ws-max-size <int>` - Set the WebSockets max message size, in bytes. Please note that this can be used only with the default `websockets` protocol.
+* `--ws-max-queue <int>` - Set the WebSockets maximum length of the queue that holds incoming messages size. Please note that this can be used only with the default `websockets` protocol.
```

Comment:
```suggestion
* `--ws-max-queue <int>` - Set the maximum length of the WebSocket incoming messages queue. Please note that this can be used only with the default `websockets` protocol.
```